tasks/release.py

```diff
@@ -1251,7 +1251,11 @@ def create_rc(ctx, major_versions="6,7", patch_version=False, upstream="origin")
     )
 
 
-@task(help={'redo': "Redo the tag & build for the last RC that was tagged, instead of creating tags for the next RC."})
+@task(
+    help={
+        'redo': "The redo functionality was removed because of dangerous --force functionality. Please build another RC."
+    }
+)
 def build_rc(ctx, major_versions="6,7", patch_version=False, redo=False):
```

Comment:
I think it's better to keep it for now, especially because it raises an error that explains what should be done instead. We can completely remove it in couple of releases once everyone involved is familiar with this change.

Reply:
Ok, I guess you're right that the amount of people using this is so limited that just removing it altogether makes sense. I'll do that.